docs/source/common_issues.rst

```diff
@@ -123,14 +123,17 @@ Types of empty collections
 --------------------------
 
 You often need to specify the type when you assign an empty list or
-dict to a new variable, as mentioned earlier:
+dict to a new variable:
 
 .. code-block:: python
 
-   a = []  # type: List[int]
+   a = []  # error: Need type annotation for variable
 
-Without the annotation mypy can't always figure out the
-precise type of ``a``.
+Without an annotation mypy can't always figure out the precise type of ``a``.
+
+.. code-block:: python
+
+   a = [] # type: List[int]
```

Comment:
I think it would make more sense to flip the two examples.

Something like the following (I'm omitting the exact words there)
```

1
--
You often need to specify the type...:

a = []  # type: List[int]

Without the annotation...

a = []  # error: Need type annotation for variable
```